airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.12.0
```

Comment:
Having to change the public method `read_config` is a breaking change in the API so I bumped the version. However, I feel like the change I made shouldn't affect the version number and maybe this means that we would benefit for better separation of concerns. For example, if the validation of input (JSON file in this case) would not be part of the domain, we would probably be shielded from breaking change in that case.

Reply:
just a heads up that I merged a patch version for a fix as `0.11.1`. Your version should be fine, but just a reminder to get the latest master to make sure our changelog is up to date